### ISSUE TITLE
Fedora 22 VM Tools not installed

### ISSUE THREAD

**Issue by mmckinst** [chef/bento#458]
see below error from /var/log/vboxadd-install.log on the fedora-22 bento box

```
grep: /lib/modules/4.0.4-301.fc22.x86_64/build/include/linux/version.h: No such file or directory
make KBUILD_VERBOSE=1 CONFIG_MODULE_SIG= -C /lib/modules/4.0.4-301.fc22.x86_64/build SUBDIRS=/tmp/vbox.0 SRCROOT=/tmp/vbox.0 modules
arch/x86/Makefile:114: stack-protector enabled but compiler support broken
Makefile:666: Cannot use CONFIG_CC_STACKPROTECTOR_STRONG: -fstack-protector-strong not supported by compiler
make[1]: gcc: Command not found
```

While gcc and cpp are in the ks.cfg, they are not actually on the Fedora-Server-DVD-x86_64-22.iso bento uses to install all the packages from. You can verify this by mounting the iso and doing a find for the packages. When the Guest Additions try to compile, it can't because it lacks gcc.

**Comment by cheeseplus** [comment]
Yep, we've encountered this with the Vmware Fusion provider first but strangely it's the only one that actually breaks a build. A PR is incoming that should fix them all and it should be a straightforward "use DNF to install packages before this part"

**Comment by cheeseplus** [comment]
Fixed efeb14a810551b06d7c5718639034cd9d3b45219
